qa/suites/rbd/encryption/workloads/qemu_xfstests_luks1.yaml

```diff
@@ -2,6 +2,22 @@ overrides:
   install:
     ceph:
       extra_packages: [rbd-nbd]
+  ceph:
+    conf:
+      osd:
+        debug bluefs: "1/5"
+        debug bluestore: "1/5"
+        debug filestore: "1/3"
+        debug journal: "0/5"
+        debug rocksdb: "4/5"
+  ceph-deploy:
```

Comment:
Is `ceph-deploy` even used anymore? We can probably simplify this a little bit by just adding it as a YAML snippet in the `clusters` directory as well, so that it applies to all workunits under the `encryption`, `migration`, and `qemu` directories.

Reply:
I have no idea about `ceph-deploy`, just saw that the config was set for it. Anyway it would most likely not do any difference. I will try with it removed.

And I will try with a YAML snippet. I did not do this initially because I would not sure I would be ok to disable logs for all workunits.

Reply:
I presumed they were the defaults [1]

[1] https://github.com/ceph/ceph/blob/master/src/common/subsys.h#L70

Reply:
Ah, of course.  I just assumed the defaults were 0/5 or 1/5 and didn't think to check...

Reply:
ceph-deploy isn't maintained or tested anymore in master

Reply:
we may want to reduce objectstore debugging globally, given it's the largest portion of the logs and bluestore is quite stable these days. What do you think @ifed01 @aclamk @neha-ojha ?

Reply:
we can get rid of the ceph-deploy part and @jdurgin agreed, for these rbd tests we any don't need bluestore related logging

Reply:
I was suggesting globally as in all the suites, not just rbd

Reply:
Sure, we don't want to add more facets with extensive bluestore logging, anything outside of rados (and perhaps upgrade) doesn't need it. Also, I'd suggest having some level of bluestore logging for https://github.com/ceph/ceph/tree/master/qa/suites/rados/objectstore, for the rest, I'll defer to @ifed01 and @aclamk.

Reply:
@neha-ojha @ifed01 @aclamk Any final objections to this plan? Otherwise, I'll open a new PR tomorrow to drop the global objectstore debug log overrides and move them to the rados and upgrade suites specifically.

Reply:
Generally I would like to have more verbose logging to troubleshoot crashes. Hence can we still use something like:
debug-bluestore = 1/20
debug-bluefs = 1/10

Reply:
@ifed01 The issue being that it's way too expensive to keep the bluestore logs at `1/X` in general and the proposal is to move those debug settings off the RBD/CephFS/RGW suites. It's not an option, at least from the RBD point-of-view, to keep the RBD suite running bluestore are `1/X`.

Reply:
> It's not an option, at least from the RBD point-of-view, to keep the RBD suite running bluestore are 1/X.

Is that really the case?  This PR set it to 1/5 (the default from `common/subsys.h`) and it would have been merged if @jdurgin hadn't mentioned the possibility of reducing objectstore debugging globally.  @ifed01is suggesting 1/20 which is the same from disk usage/archiving perspective -- and that was our primary concern, so it seems acceptable to me.

For performance sensitive or benchmark type jobs we would need explicit 0/0 to override the default anyway.

Reply:
D'oh -- I was looking at your change and not the current overrides which are `20/20`. Disregard my comment and I'll open a PR today.